src/renderer/components/Settings.tsx

```diff
@@ -1,16 +1,31 @@
 import { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom'; // Use useNavigate instead of useHistory
+import { changeUser, updateUser, selectUser } from '../../redux/userSlice';
 import { services } from '../../services';
-import useSignalRHub from '../hooks/useSignaRHub';
-import { Role, Studio } from '../../types';
+import { Role, Studio, User } from '../../types';
 import { enumNumericValues } from '../../utils/utils';
+import { useAppDispatch } from '../hooks/appStore';
+import { useSelector } from 'react-redux';
 
-function AssignStudioToUser() {
+interface RoleOptions {
+  name: string;
+  role: Role;
+}
+
+function Settings() {
   const [studios, setStudios] = useState<Studio[]>([]);
   const [selectedStudio, setSelectedStudio] = useState<Studio | null>(null);
-  const [selectedRole, setSelectedRole] = useState<Role | null>(null);
+  const [selectedRole, setSelectedRole] = useState<RoleOptions | null>(null);
   const [deviceId, setDeviceId] = useState('');
   const navigate = useNavigate();
+  const dispatch = useAppDispatch();
+  const currentUser = useSelector(selectUser);
+
+  const rolesOptions = [
```

Comment:
Not needed, just build two buttons based on enum values (Team, Host). In future we will have n-teams. And we will calculate team number on backend.